Sources/Plasma/NucleusLib/inc/pnSingletons.cpp

```diff
@@ -43,7 +43,9 @@ You can contact Cyan Worlds, Inc. by email legal@cyan.com
 #include "plgDispatch.h"
 #include "hsResMgr.h"
 #include "plPipeResReq.h"
+#include "plPipeline.h"
 
+plDisplayHelper* plDisplayHelper::_currentDisplayHelper = nullptr;
```

Comment:
Feels really weird sticking this in NucleusLib, but that's where plPipeline lives, so maybe it's okay?

pl3DPipeline in PubUtilLib might be a better spot

Reply:
Agreed - the null pipeline probably doesn't care about displays